lms/envs/common.py

```diff
@@ -1019,6 +1019,18 @@
     # .. toggle_target_removal_date: None
     # .. toggle_tickets: 'https://openedx.atlassian.net/browse/MST-1458'
     'ENABLE_CERTIFICATES_IDV_REQUIREMENT': False,
+
+    # .. toggle_name: FEATURES['ENABLE_MFE_API']
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: False
+    # .. toggle_description: Set to True to enable MFE Config REST API. This is disabled by
+    #   default.
+    # .. toggle_use_cases: open_edx
+    # .. toggle_creation_date: 2022-05-20
+    # .. toggle_target_removal_date: None
+    # .. toggle_warnings: None
+    # .. toggle_tickets: None
+    'ENABLE_MFE_API': False,
```

Comment:
The features dictionary is kind of historic and I don't think it makes sense to add this to the features dict.  I think you should just make it a top level setting.

Reply:
Ready, thanks for the feedback Feanil 🎉